melodic/distribution.yaml

```diff
@@ -2794,6 +2794,14 @@ repositories:
         release: release/melodic/{package}/{version}
       url: https://github.com/wxmerkt/exotica_val_description-release.git
       version: 1.0.0-1
+  extest:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/nakakura/extest-release.git
```

Comment:
The top repository does not reflect the LICENSE version of the package

Reply:
The MIT license is declared in their package.xml file but I would agree it would be better to include the MIT license text in the repository @nakakura.

Reply:
@Blast545  @nuclearsandwich  Sure! Sorry I forgot to include it.